docs/modules/data-structures/pages/vector-search-overview.adoc

```diff
@@ -93,12 +93,24 @@ To allow the system to return enough results, the following conditions must be s
 
 - `partitionLimit * partitionCount >= topK`, `partitionLimit &lt;= topK`
 - `memberLimit * memberCount >= topK`, `memberLimit &lt;= topK`
+- `efSearch >= partitionLimit`, if `partitionLimit` is not configured explicitly this applies to the default `partitionLimit` value
```

Comment:
AFAIS the default value of `partitionLimit` is not mentioned anywhere.

Reply:
the exact formula is complicated (https://github.com/hazelcast/hazelcast-mono/pull/3258). I described it generally:

> `partitionLimit` is calculated based on `topK` and cluster configuration (number of partitions)